beacon_node/network/src/network_beacon_processor/gossip_methods.rs

```diff
@@ -241,6 +254,21 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         packages: GossipAttestationBatch<T::EthSpec>,
         reprocess_tx: Option<mpsc::Sender<ReprocessQueueMessage>>,
     ) {
+        for package in &packages {
```

Comment:
You can move attestation metrics to `process_gossip_attestation_result` and only track valid objects